tools/build/config/rollup.config.mjs

```diff
@@ -116,6 +116,8 @@ const external = [
   /^highlight/,
   /^lodash\//,
   /^react/,
+  'jscodeshift/src/Runner',
+  'is-git-clean',
```

Comment:
I'm not sure that I need this here.

> Definition: externals in Rollup refer to modules that are treated as external dependencies. These modules are not included in the bundle but are expected to be available in the environment where the bundle is executed.
> 
> Usage: You specify externals in your Rollup configuration to prevent certain modules from being bundled. This is particularly useful when you want to avoid duplicating dependencies that are already present in the target environment (e.g., a browser, Node.js, or another bundler).

Reply:
I don't think these should be here. Otherwise, iirc, this would require consuming apps to install `jscodeshift` as a dependency